src/Correlate.Core/CorrelationManager.cs

```diff
@@ -205,12 +212,12 @@ private T Execute<T>(string? correlationId, Func<T> correlatedFunc, OnException?
         }
     }
 
-    private static bool HandlesException<T>(OnException? onException, CorrelationContext correlationContext, Exception ex, out T result)
+    private bool HandlesException<T>(OnException? onException, CorrelationContext correlationContext, Exception ex, out T result)
     {
-        if (!ex.Data.Contains(CorrelateConstants.CorrelationIdKey))
+        if (!ex.Data.Contains(_options.LoggingScopeKey))
         {
             // Because we're about to lose context scope, enrich exception with correlation id for reference by calling code.
-            ex.Data.Add(CorrelateConstants.CorrelationIdKey, correlationContext.CorrelationId);
+            ex.Data.Add(_options.LoggingScopeKey, correlationContext.CorrelationId);
```

Comment:
Revert this change as it has nothing to do with log scope itself, even though the comment hints the relation to it. It is exception metadata.

Reply:
Reverted exception data to always contain `CorrelationId` property.